cpp/src/arrow/flight/sql/odbc/odbc_api.h

```diff
@@ -33,4 +33,8 @@ namespace arrow
 {
   SQLRETURN SQLAllocHandle(SQLSMALLINT type, SQLHANDLE parent, SQLHANDLE* result);
   SQLRETURN SQLFreeHandle(SQLSMALLINT type, SQLHANDLE handle);
-} // namespace arrow
+  SQLRETURN SQLGetEnvAttr(SQLHENV env, SQLINTEGER attr, SQLPOINTER valuePtr,
+                          SQLINTEGER bufferLen, SQLINTEGER* strLenPtr);
+  SQLRETURN SQLSetEnvAttr(SQLHENV env, SQLINTEGER attr, SQLPOINTER valuePtr,
+                          SQLINTEGER strLen);
+  } // namespace arrow
```

Comment:
```suggestion
} // namespace arrow
```
nit

Reply:
Should run clang-format to be compliant with arrow coding standards as part of the PR process.

Reply:
@jduo you're right, I have re-enabled the lint check, but looks like we have ran out of GitHub Action minutes for the lint check. So we will need to run the checks locally for now.